test/integration/buyers/users_controller_integration_test.rb

```diff
@@ -68,4 +68,15 @@ def setup
     assert_equal 'newemail@example.org', user.email
     assert_equal 'Japan', user.field_value('country')
   end
+
+  test "#activate" do
```

Comment:
Was this action calling some issue with onboardings @akostadinov ? What was that?

Reply:
Yes, it is creating an onboarding. But because  onboardings had a wrong db trigger, it caused a tenant_id leak error.